apis/actions.md

```diff
@@ -51,31 +51,32 @@ If you compile your app, you can see that it now has a custom HeaderBar with a b
 
 ## GLib.SimpleAction
 
-Let's define a new Quit action by adding the following to the beginning of the `activate ()` method
+Let's define a new Quit action and register it with `Application` from inside the `startup` method:
 
 ```vala
-var quit_action = new SimpleAction ("quit", null);
+public override void startup () {
+    base.startup ();
 
-add_action (quit_action);
-set_accels_for_action ("app.quit",  {"<Control>q", "<Control>w"});
-```
-
-and this to the end of the `activate ()` method:
+    var quit_action = new SimpleAction ("quit", null);
 
-```vala
-quit_action.activate.connect (() => {
-    main_window.destroy ();
-});
+    add_action (quit_action);
+    set_accels_for_action ("app.quit",  {"<Control>q", "<Control>w"});
+    quit_action.activate.connect (quit);
+}
 ```
 
 You'll notice that we do a few things here:
 
 * Instantiate a new [`GLib.SimpleAction`](https://valadoc.org/gio-2.0/GLib.SimpleAction.html) with the name "quit"
 * Add the action to our `Gtk.Application`'s [`ActionMap`](https://valadoc.org/gio-2.0/GLib.ActionMap.html)
 * Set the "accelerators" \(keyboard shortcuts\) for "app.quit" to `<Control>q` and `<Control>w"`. Notice that the action name is prefixed with `app`; this refers to the `ActionMap` built in to `Gtk.Application`
-* Connect to the `activate` signal of our `SimpleAction` and call `destroy ()` on `main_window`. This must be at the end of `activate ()` because of that reference to `main_window`
+* Connect the `activate` signal of our `SimpleAction` to Application's [quit method](https://valadoc.org/gio-2.0/GLib.Application.quit.html).
 
-and now we can tie the action to the HeaderBar Button by assigning the `action_name` property of our Button:
+{% hint style="info" %}
```

Comment:
I wonder if we should swap this with the hint on line 88/89 since that hint is more immediately related to this section and a "learn more" hint is probably most useful after we finish here and not before we're done

Reply:
I've switched the 2 info boxes in #184 and #185.